ports/atmel-samd/boards/seeeduino_xiao/pins.c

```diff
@@ -44,9 +44,17 @@ STATIC const mp_rom_map_elem_t board_module_globals_table[] = {
     { MP_ROM_QSTR(MP_QSTR_SCL), MP_ROM_PTR(&pin_PA09) },
     { MP_ROM_QSTR(MP_QSTR_SDA), MP_ROM_PTR(&pin_PA08) },
 
-    // LED pins
-    { MP_ROM_QSTR(MP_QSTR_LED),   MP_ROM_PTR(&pin_PA17) }, // status
-    { MP_ROM_QSTR(MP_QSTR_BLUE_LED),   MP_ROM_PTR(&pin_PA17) },
+    // LED pins  ... note XIAO is wired so high/1/True turns LED OFF
+    //                                     low/0/False turns LED ON
+    //    duplicate names for test LED ... see D13
+    { MP_ROM_QSTR(MP_QSTR_LED),   MP_ROM_PTR(&pin_PA17) },
+    { MP_ROM_QSTR(MP_QSTR_YELLOW_LED),   MP_ROM_PTR(&pin_PA17) },
+    //    2 blue LEDs ... uart rx/tx indicator is default use
+    { MP_ROM_QSTR(MP_QSTR_RX_LED),   MP_ROM_PTR(&pin_PA18) },
+    { MP_ROM_QSTR(MP_QSTR_TX_LED),   MP_ROM_PTR(&pin_PA19) },
+    //    create duplicate mappings with BLUE in name
+    { MP_ROM_QSTR(MP_QSTR_BLUE1_LED),   MP_ROM_PTR(&pin_PA18) },
+    { MP_ROM_QSTR(MP_QSTR_BLUE2_LED),   MP_ROM_PTR(&pin_PA19) },
```

Comment:
1. Group aliases for the same pin together, with blank lines in between group.
2. All these LED pins are active-low, so we add the `_INVERTED` suffix on such pins. (So there is no plain `board.LED` any more.)
3. You can omit the comments because the names are pretty self-explanatory.
4. We have tended to add a number after the word `LED`

To see all the LED names in use (not completely consistent):
```sh
grep _LED ports/*/boards/*/pins.c
```

```suggestion
    { MP_ROM_QSTR(MP_QSTR_LED_INVERTED),   MP_ROM_PTR(&pin_PA17) },
    { MP_ROM_QSTR(MP_QSTR_YELLOW_LED_INVERTED),   MP_ROM_PTR(&pin_PA17) },
    
    { MP_ROM_QSTR(MP_QSTR_RX_LED_INVERTED),   MP_ROM_PTR(&pin_PA18) },
    { MP_ROM_QSTR(MP_QSTR_BLUE_LED1_INVERTED),   MP_ROM_PTR(&pin_PA18) },

    { MP_ROM_QSTR(MP_QSTR_TX_LED_INVERTED),   MP_ROM_PTR(&pin_PA19) },
    { MP_ROM_QSTR(MP_QSTR_BLUE_LED2_INVERTED),   MP_ROM_PTR(&pin_PA19) },
```